docs/content/developer/getting-started/iota-environment.mdx

```diff
@@ -34,6 +34,4 @@ IOTA provides two official SDKs that you can use to interact with the IOTA netwo
 
 ## Move IDEs and plugins
 
-We recommend that you use the [Visual Studio Code](https://code.visualstudio.com/) IDE.
-
-We are currently developing a code analyzer plugin for Visual Studio Code - details to follow soon.
\ No newline at end of file
+We recommend that you use the [Visual Studio Code](https://code.visualstudio.com/) IDE with the [IOTA Move](https://marketplace.visualstudio.com/items?itemName=iotaledger.iota-move) extension installed.
```

Comment:
This link 404s, is the URL correct?

Reply:
Yes, it is correct. The extension was published and removed, and it will be republished soon

Reply:
@lucas-tortora it is published.